src/CoreBundle/Resources/config/form_types.xml

```diff
@@ -23,7 +23,12 @@
             <tag name="form.type" alias="sonata_type_datetime_range"/>
             <argument type="service" id="translator"/>
         </service>
-        <service id="sonata.core.form.type.date_picker" class="Sonata\CoreBundle\Form\Type\DatePickerType" public="true">
+        <service id="sonata.core.form.type.date_picker_legacy" class="Sonata\CoreBundle\Form\Type\DatePickerType" public="true">
+            <tag name="form.type" alias="sonata_type_date_picker"/>
```

Comment:
I'm using the same alias here, no idea if this will actually work…

Reply:
The 2.8 UPGRADE guide says to remove it :sweat_smile:  : https://github.com/symfony/symfony/blob/2.8/UPGRADE-2.8.md#form

Reply:
can we use the same alias at 2 places?

Reply:
I have no idea

Reply:
isn't alias used for < sf 2.8? Now using class is the recommended thing

Reply:
Maybe we can remove them all…